docs/useful-snippets/snippets.rst

```diff
@@ -58,3 +58,56 @@ code as follows.
 If you would still like to run your code in Jupyter you will need to adjust the cell as noted above. Then you can
 you can add the `nbmultitask library <(https://nbviewer.org/github/micahscopes/nbmultitask/blob/39b6f31b047e8a51a0fcb5c93ae4572684f877ce/examples.ipynb)>`__
 or look at this `stackoverflow <https://stackoverflow.com/questions/50937362/multiprocessing-on-python-3-jupyter>`__.
+
+Using multiple schedules and data collectors
+-------------
+
+One may occasionally want multiple schedules with distinct sets of agents and collect data on each of those schedules independently while allowing the agents to interact in the same model. This would be particularly useful if the agents in your model do not all have the same attributes and you want to collect data on them in groups. You can create a simple scheduler class to mix the agents in each schedule and data collect on each schedule independently.
+
+.. code:: python
+
+  from random import shuffle
+
+  # simple schedule mixer
+  class ScheduleMixerRandom:
+      """
+      Note that ScheduleMixerRandom as written here would not capture changes to
```

Comment:
Replacing "note that" with "be careful:" would better alarm me as a reader, of the consequence of not updating `ScheduleMixerRandom._agents` together with the schedules.

Reply:
Adjusted both.